ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDashboard.robot

```diff
@@ -99,7 +99,7 @@ Wait For RHODS Dashboard To Load
     ...          ${expected_page}=${NONE}    ${timeout}=60
     ${half_timeout}=   Evaluate    int(${timeout}) / 2
     Wait For Condition    return document.title == ${dashboard_title}    timeout=${half_timeout}
-    Wait Until Page Contains Element    xpath:${RHODS_LOGO_XPATH}    timeout=${half_timeout}
+    Wait Until Page Contains Element    xpath:${RHODS_LOGO_XPATH}    timeout=${timeout}
```

Comment:
now this waits for timeout + half_timeout that is given to this keyword... this can be confusing for someone